examples/language/lib/class_modifiers/ex4/a.dart

```diff
@@ -0,0 +1,6 @@
+// Library a.dart
+final class Vehicle {
+  void moveForward(int meters) {
+    //...
```

Comment:
To be consistent with the other ellipsis comments:

```suggestion
    // ...
```